.github/workflows/ci.yml

```diff
@@ -1034,33 +1035,43 @@ jobs:
           echo "can_tag output is $CAN_TAG"
           echo "can_tag=$CAN_TAG" >> $GITHUB_OUTPUT
 
-  gaugerelease:
-    name: Check unreleased changes
+  dispatchrelease:
+    name: Dispatch tag release
     runs-on: ubuntu-latest
     needs: [tests, checkgovernance]
-    if: ${{ needs.checkgovernance.outputs.can_tag == '1' && (github.event_name == 'push' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') }}
-    outputs:
-      do_release: ${{ steps.gauge-release.outputs.do_release }}
-      next_tag: ${{ steps.gauge-release.outputs.next_tag }}
+    if: ${{ needs.checkgovernance.outputs.can_tag == '1' }}
+    env:
+      GITHUB_REPOSITORY: ${{ github.repository }}
+      BRANCH: ${{ github.ref_name }}
     steps:
-      - name: Gauge release
-        id: gauge-release
-        uses: silverstripe/gha-gauge-release@v1
-        with:
-          latest_local_sha: ${{ needs.tests.outputs.latest_local_sha }}
 
-  patchrelease:
-    name: Patch release
-    runs-on: ubuntu-latest
-    needs: gaugerelease
-    if: ${{ needs.gaugerelease.outputs.do_release == '1' }}
-    permissions:
-      contents: write
-    steps:
-      - name: Patch release
-        uses: silverstripe/gha-tag-release@v1
-        with:
-          tag: ${{ needs.gaugerelease.outputs.next_tag }}
-          delete_existing: false
-          release: true
-          release_auto_notes: true
+      - name: Dispatch release
+        shell: bash
+        id: dispatch-release
```

Comment:
```suggestion
      - name: Dispatch tag patch release
        shell: bash
        id: dispatch-tag-patch-release
```

Reply:
Could probably just delete `id: dispatch-release` - I don't think it's used for anything

Reply:
I thought smaller names were better? I'll leave as is 'cause this matches the name of the job and the names in gha-action-ci.
If you want them changed, they should still say "dispatch" (because that's what this is doing) and should be the same across the board.

Reply:
I would have thought descriptive is better, it matches the workflow that's being dispatched.  We have many actions/workflows, many of which sound similar, so seems better to use the exact name, easier to work out what's going on from a glance

> If you want them changed, they should still say "dispatch" (because that's what this is doing) and should be the same across the board.

Yeah "Dispatch tag patch release" that's what my recommendation is

Reply:
> I would have thought descriptive is better

Usually I agree, but don't longer names get cut off? I'm not too fussed though, I'll make the change.

Reply:
Done